trunk/README.txt

```diff
@@ -158,6 +158,10 @@ Filter the regular expression used to find URLs to be converted to https://go.pa
 
 == Changelog ==
 
+= 1.5.8 =
+
+* Fix - Handled an issue that authentication failure causes the settings page stuck in loading
```

Comment:
would you mind linking to the 184 error code page so we are being clear on exactly what auth failure